### PR TITLE
docs(k8s): making it clear users MUST update secrets for prod instances.

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

Addressing some confusion from this issue: https://github.com/apache/superset/issues/27931

### ADDITIONAL INFORMATION

- [ ] Has associated issue: Fixes https://github.com/apache/superset/issues/27931
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by rusackas** [comment]
@margorczynski  let me know if this addresses your concern, or make suggestions accordingly if not :) Thanks.

File: docs/docs/installation/running-on-kubernetes.mdx [hunk 1]
```diff
@@ -93,7 +93,7 @@ Depending how you configured external access, the URL will vary. Once you've ide
 
 #### Security settings
 
-Default security settings and passwords are included but you **SHOULD** override those with your own, in particular:
+Default security settings and passwords are included but you **SHOULD** override those with your own for `dev` instances, and **MUST** update them to run `prod` instances, in particular:
```
Comment (villebro) [inline docs/docs/installation/running-on-kubernetes.mdx:96]:
```suggestion
Default security settings and passwords are included but you **MUST** update them to run `prod` instances, in particular:
```

Maybe this is simpler - we already log a warning when starting in debug with the default `SECRET_KEY`, so people should see that whenever they start their local devenv without a `SECRET_KEY` of their own.

**Review by villebro — APPROVED** [review]
LGTM with an alternate simplified version.

File: docs/docs/installation/running-on-kubernetes.mdx (hunk 1 above)
Comment (rusackas) [reply docs/docs/installation/running-on-kubernetes.mdx]: Works for me! Thanks.

**Review by mistercrunch — APPROVED** [review]

**Comment by margorczynski** [comment]
Hey @rusackas, yep, I think this more stronly suggests that the config needs to be overwritten for this to work, thank you for the update :)

**Pull request merged**